docs/tbdev_getting_started.ipynb

```diff
@@ -12,7 +12,7 @@
     },
```

Comment:
<p>Is there an extra newline here before "In the above"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/3920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
This will not render in the notebook
![image](https://user-images.githubusercontent.com/547150/89841120-20a07d00-db40-11ea-88bb-0394a8d73adf.png)